runhouse/main.py

```diff
@@ -610,138 +610,140 @@ def get_clusters_from_den():
 
-        last_active_at = den_cluster.get("status_last_checked")
-        last_active_at = (
-            datetime.datetime.fromisoformat(last_active_at.split(".")[0])
-            if isinstance(last_active_at, str)
-            else None
+def _print_msg_live_sky_clusters(sky_clusters: int):
+    if sky_clusters > 0:
+        console.print(
+            f"There are {sky_clusters} live cluster(s) that are not saved in Den. For more information, please run [bold italic]sky status -r[/bold italic]."
         )
-        last_active_at = (
-            last_active_at.replace(tzinfo=datetime.timezone.utc)
-            if last_active_at
-            else None
+
+
+def supported_time_units():
+    return ", ".join(list(TIME_UNITS.keys()))
+
+
+@cluster_app.command("list")
+def cluster_list(
+    show_all: bool = typer.Option(
+        False,
+        "--a",
+        "--all",
+        help="Get all den clusters. Up to 50 most recently active clusters will be displayed.",
+    ),
+    since: Optional[str] = typer.Option(
+        None,
+        "--since",
+        help="Time duration to filter on. Minimum time to filter on is 1 minute. If the provided value will be less "
+        "than that, the filter will be set to 1 minute by default. You may filter by seconds (s), "
+        "minutes (m), hours (h) or days (s). Examples: 30s, 15m, 2h, 3d.",
+    ),
+    cluster_status: Optional[str] = typer.Option(
+        None,
+        "--status",
+        help="Cluster status to filter on. Supported filter values: running, terminated, runhouse_daemon_down.",
+    ),
+):
+    """Load Runhouse clusters"""
+
+    # logged out case
+    if not rh.configs.token:
+        # TODO [SB]: adjust msg formatting (coloring etc)
+        sky_cli_command_formatted = f"{italic_bold_ansi}sky status -r{reset_format}"  # will be printed bold and italic
+        console.print(
+            f"This feature is available only for Den users. Please run {sky_cli_command_formatted} to get on-demand cluster(s) information or sign-up Den."
         )
+        return
 
-        if cluster_status == "running" and not last_active_at:
-            # For BC, in case there are clusters that were saved and created before we introduced sending cluster status to den.
-            cluster_status = "unknown"
+    clusters = Cluster.list(show_all=show_all, since=since, status=cluster_status)
 
-        cluster_info = {
-            "Name": cluster_name,
-            "Cluster Type": cluster_type,
-            "Status": cluster_status,
-            "Last Active (UTC)": last_active_at,
-        }
-        running_clusters.append(
-            cluster_info
-        ) if cluster_status == "running" else not_running_clusters.append(cluster_info)
+    all_clusters = clusters.get("all_clusters", None)
+    running_clusters = clusters.get("running_clusters", None)
+    sky_clusters = clusters.get("sky_clusters", None)
+
+    if not all_clusters:
+        console.print("No existing clusters.")
```

Comment:
can we avoid printing an empty table in this scenario? 
and the header that we show (Running: 0/0, Total Displayed), etc. can also be removed in this case